frontend/src/constants/endpoints.ts

```diff
@@ -1,5 +1,5 @@
 export const API_ENDPOINTS = {
-  contactUs: '/contact-us',
-  newsletter: '/newsletter',
-  allEvents: '/events',
+  'contact-us': '/contact-us',
+  'newsletter': '/newsletter',
+  'events': '/events',
```

Comment:
Any specific reason to put them inside `' '` ? I think it shouldn't be creating any problems. 🤔

Reply:
contact-us was creating issue without `' '` so to keep the consistencies

Reply:
Event though its working, I don't think this is correct way to do it in javascript objects. 🤔  @Saurabhsharma884 views?

Reply:
please direct.

Reply:
Putting inverted commas in javascript object keys is actually not considered good practice. As per the JS naming conventions the recommended way is to use `camelCase` for naming variables .Even if the object keys are defined without quotes, they are automatically converted to strings when accessed or used. We usually keep keys in inverted commas in JSON and so we can omit the inverted commas while working with JS.

Reply:
thank you for the knowledge. I have updated